ckanext/s3filestore/tasks.py

```diff
@@ -9,7 +9,7 @@
 log = logging.getLogger(__name__)
 
 
-def s3_afterUpdatePackage(ckan_ini_filepath, visibility_level, pkg_id, pkg_dict):
+def s3_afterUpdatePackage(ckan_ini_filepath, visibility_level, pkg_id, pkg_dict=None):
```

Comment:
There should probably be a note that the parameter is deprecated.

Reply:
done, also removed ckan ini filepath as archiver proved you don't need it